src/mkdir.js

```diff
@@ -83,16 +67,23 @@ function _mkdir(options, dirs) {
         fs.mkdirSync(dir, parseInt('0777', 8));
       }
     } catch (e) {
-      var reason;
-      if (e.code === 'EACCES') {
-        reason = 'Permission denied';
-      } else if (e.code === 'ENOTDIR' || e.code === 'ENOENT') {
```

Comment:
I think `ENOENT` should be handled distinctly from `ENOTDIR`, not sure why they're in the same branch here.

Reply:
Hmmm this is weird. On unix, ENNOTDIR and ENOENT are for different reasons. On appveyor, I see ENOENT when part of the path is a regular file. Not sure yet how we should handle this.

Reply:
Is this still causing the appveyor CI to fail?

Reply:
Yes. I haven't fixed this yet.